skimage/draw/draw.py

```diff
@@ -475,8 +485,27 @@ def polygon(r, c, shape=None):
            [0, 0, 0, 0, 1, 1, 0, 0, 0, 0],
            [0, 0, 0, 0, 1, 0, 0, 0, 0, 0],
            [0, 0, 0, 0, 1, 0, 0, 0, 0, 0],
-           [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
+           [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]])
+
+    If the image `shape` is defined and vertices / points of the `polygon` are
+    outside this coordinate space, only a part (or none at all) of the polygon's
```

Comment:
If this example is meant to demonstrate the offset, then what the offset is doing should be mentioned here. I would suggest replacing with:

Defining an image `shape` can be used to draw an area of the polygon smaller or larger than the extent of the polygon's vertices. This example shows how shifting the polygon's vertices by an offset can be used to draw an arbitrary sub-region of the polygon

Reply:
Great suggestion! I found my first sentence clearer but incorporated your second one in 6221476b7c67d677787fdc83418e8d67b2dfd23f. :)